.travis.yml

```diff
@@ -23,12 +23,12 @@ matrix:
       env: SYMFONY_LTS='^2'
     # Symfony 3 LTS
     - php: 7.0
-      env: SYMFONY_LTS='^3' DEPENDENCIES='dev'
+      env: SYMFONY_LTS='^3'
     # development dependencies
     - php: 7.1
-      env: DEPENDENCIES='dev'
```

Comment:
We should keep this job, but add a dedicated job that uses PHP 7.1 with stable dependencies (it will automatically use Symfony 4 then right now).